src/services/item/plugins/app/chatBot/schemas.ts

```diff
@@ -1,33 +1,39 @@
+import { Type } from '@sinclair/typebox';
+import { StatusCodes } from 'http-status-codes';
+
 import { ChatbotRole, GPTVersion } from '@graasp/sdk';
 
-const chatBotDefinition = {
-  type: 'object',
-  required: ['role', 'content'],
-  properties: {
-    role: { type: 'string', enum: Object.values(ChatbotRole) },
-    content: { type: 'string' },
-  },
-};
+import { customType } from '../../../../../plugins/typebox';
+import { errorSchemaRef } from '../../../../../schemas/global';
 
-const create = {
-  body: {
-    type: 'array',
-    items: chatBotDefinition,
-    minItems: 1, // do not allow empty array
+const chatBotDefinition = Type.Object(
```

Comment:
This could be called a history of the conversation. Not sure calling it `chatbotDefinition` is appropriate.